documentation/docs/developers/strategies/twap.md

```diff
@@ -0,0 +1,53 @@
+# TWAP
+
+Unlike the previous tutorial strategies which demonstrated straight-forward tasks like adding configurations and using clock ticks to periodically perform action, this strategy will introduce you to implement a more complicated algorithm. The tools demonstrated in this strategy is : 
+1) Maintaining some states over clock ticks to retain important information that will decide algorithm's next action 
+2) Quantizing order size depending of user's specification of how the user wants order to be executed.
```

Comment:
Quantizing order sizes is needed in a lot of situations, apart from just "user's specification". I'd recommend just simplify this point to "Quantizing order sizes"